docs/bootc-images.md

```diff
@@ -55,4 +55,79 @@ as part of your `RUN` invocations.   This will perform early detection
 of some incompatibilities but is not a strict requirement today and will not be
 in the future.
 
+# Building and using `bootc` compatible images in the real world
```

Comment:
I understand the sentiment but...overall ISTM we should try to improve the existing bits of the docs to match the "real world"